dev-ostelco-ios-clientTests/APITests/MockAPITests.swift

```diff
@@ -226,4 +330,48 @@ class MockAPITests: XCTestCase {
             XCTFail("Unexpected error: \(error)")
         }
     }
+    
+    func testMockCreatingAddress() {
+        let address = EKYCAddress(street: "123 Fake Street",
+                                  unit: "3",
+                                  city: "Fake City",
+                                  postcode: "12345",
+                                  country: "Singapore")
+        
+        OHHTTPStubs.stubRequests(passingTest: isAbsoluteURLString("https://api.fake.org/regions/sg/kyc/profile?address=123%20Fake%20Street;;;3;;;Fake%20City;;;12345;;;Singapore&phoneNumber=12345678") && isMethodPUT(), withStubResponse: { _ in
+            // TODO: Figure out why this is a `204 No Content` instead of a `201 Created` on the real API
+            return OHHTTPStubsResponse(data: Data(), statusCode: 204, headers: nil)
+        })
+        
+        // Failure handled in `awaitResult`
+        self.mockAPI.addAddress(address, forRegion: "sg").awaitResult(in: self)
+    }
+    
+    func testMockCreatingJumioScan() {
+        self.stubPath("regions/sg/kyc/jumio/scans", toLoad: "create_jumio_scan")
+        
+        guard let scan = self.mockAPI.createJumioScanForRegion(code: "sg").awaitResult(in: self) else {
```

Comment:
I thought the fix for createJumioScanForRegion was in this PR. This is using `GET` instead of `POST`

Reply:
Use this for reference: https://github.com/ostelco/ostelco-core/blob/develop/prime/infra/dev/prime-customer-api.yaml

Reply:
@prasanthu It [is in this PR](https://github.com/ostelco/ostelco-ios-client/pull/200/files#diff-7a5823cfae457badcd2f8a8fb5087f4bR2130) and it's using `POST` now